src/models/user/user.schema.ts

```diff
@@ -93,7 +93,7 @@ const UserSchema: Schema = new Schema(
     },
   },
   {
-    timestamps: {},
+    timestamps: true,
```

Comment:
I think we may need to update a bunch of these timestamp properties in the various schema

Reply:
Is it for better code, having an explicit true rather then a truthy?

We could do this as a separate issue/PR, it might be as easy as Ctrl+shift+f in Vscode to find and replace all `timestamps: {}`

Reply:
> Is it for better code, having an explicit true rather then a truthy?
> 
> We could do this as a separate issue/PR, it might be as easy as Ctrl+shift+f in Vscode to find and replace all `timestamps: {}`

I don't think mongoose's automatic timestamp functionality works without this flag set to true. I noticed that every time the record gets updated, updatedAt and createdAt are the same value. 

I might be wrong. But since I set this to true, the values seem to be updated correctly.

Reply:
I wasn't sure if you were taking care of this with your redux refactor. Otherwise yeah, let's create a separate ticket for this.

Reply:
🆒 sounds like a bit of a bug so will make for a good issue. I'm happy to PR this tomorrow (almost 🛌 ⏲️ here)